README.md

```diff
@@ -1,6 +1,6 @@
-# Polkadot Ethereum Bridge
+# Snowbridge
 
-Components for a Polkadot Ethereum Bridge
+A trust-minimized bridge between Polkadot and Ethereum.
```

Comment:
Can we say „trustless“ instead? 

I know that there is always smart contract risk and rising the consensus protocols and light client protocols involved, but even if not technically correct, the term “trustless” seems to have become the accepted way to describe that.

Reply:
sure, fixed in latest commit.